src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const personPartnerType = sex === 'f' ? 'wife' : 'husband';
+
+  const personPartner = isMarried
+    ? `${partnerName} is my ${personPartnerType}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && <p className="Person__age">I am {age}</p>}
+      {isMarried ? personPartnerType : personPartner}
```

Comment:
Add the tag `p` with this class as wrapper 
<img width="256" alt="image" src="https://github.com/user-attachments/assets/c258cb07-bf93-4443-abb3-a4f707a2ffe0">